packages/stream-model-instance-handler/src/schema-utils.ts

```diff
@@ -1,6 +1,12 @@
 import ajv, { SchemaObject } from 'ajv/dist/2020.js'
 import addFormats from 'ajv-formats'
 
+export function isEmpty(obj: any): boolean {
```

Comment:
this would return `false` if given `null` - is that the desired behavior?

Reply:
I actually feel like we **shouldn't** allow setting to an empty document `{}` but only allow setting to an explicit null.  That is a more clear signal that the goal is to delete the stream.  Allowing an empty document could let through an application programming bug where the content object is built programmatically but winds up with no fields set.  Making the app explicitly set null means its less likely to happen by accident.